include/tscpp/util/ts_errata.h

```diff
@@ -40,5 +40,16 @@ static constexpr std::array<swoc::TextView, 9> Severity_Names{
   {"Diag", "Debug", "Status", "Note", "Warn", "Error", "Fatal", "Alert", "Emergency"}
 };
 
+inline std::error_code
+ec_for()
+{
+  return {errno, std::system_category()};
+}
+inline std::error_code
+ec_for(int err)
+{
+  return {err, std::system_category()};
+}
+
```

Comment:
Nit: the name `ec_for` isn't very intuitive. IMHO naming this consistent with `make_error_code` would be more easily understandable.

Reply:
Changed to `make_errno_code`.